iib/workers/tasks/build.py

```diff
@@ -156,34 +156,40 @@ def _finish_request_post_build(
         index image.
     :param str overwrite_from_index_token: the token used for overwriting the input
         ``from_index`` image.
+    :return: pull spec of the index_image
+    :rtype: str
     :raises IIBError: if the manifest list couldn't be created and pushed
     """
     conf = get_worker_config()
+    state_reason = 'Updated the index image'
```

Comment:
I would appreciate any better suggestions here, is that state_reason okay for the else branch?

Reply:
It would be nice to add the `output_pull_spec` to the state_reason for the `else` clause.